pkg/kubelet/rkt/rkt.go

```diff
@@ -2367,26 +2429,31 @@ func (r *Runtime) GetPodStatus(uid kubetypes.UID, name, namespace string) (*kube
 		}
 	}
 
-	// If we are running no-op network plugin, then get the pod IP from the rkt pod status.
-	if r.network.PluginName() == network.DefaultPluginName {
-		if latestPod != nil {
+	if latestPod != nil {
+		// If we are running no-op network plugin, then get the pod IP from the rkt pod status.
+		if r.network.PluginName() == network.DefaultPluginName {
 			for _, n := range latestPod.Networks {
 				if n.Name == defaultNetworkName {
 					podStatus.IP = n.Ipv4
 					break
 				}
 			}
+		} else {
```

Comment:
nit: Can just return in the above `break`, so the `else` can be removed. But it's ok to keep it here as it's not big.